Bottom: 3ff87638555ade345486c3fbfb01009cd769bbd7
Top:    7178d06c2557e29b583fad75f7cc4f205114cd8a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-15 01:46:48 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index b1bf5f0..b5a7cc7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -182,7 +182,7 @@ void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
